docs/guides/page-setup.md

```diff
@@ -0,0 +1,359 @@
+
+If you do not specify margins for all sides in the margin dictionary, the old
+margins will remain in effect for the unset sides. To prevent this and set all
+remaining margins to a common size, you can use the `rest` argument. For
+example, `#set page(margin: (left: 1.5in, rest: 1in))` will set the left margin
+to 1.5 inch and the remaining margins to one inch.
+
+### Different margins on alternating pages
+Sometimes, you'll need to alternate horizontal margins for even and odd pages,
+for example to have more room towards the spine of a book than on the outsides
+of its pages. Typst keeps track of whether a page is to the left or right of the
+binding. You can use this information and set the `inside` or `outside` keys of
+the margin dictionary. The `inside` margin points towards the spine, the
+`outside` margin points towards the edge of the bound book.
+
+```example
+#set page(margin: (
+  inside: 2.5cm,
+  outside: 2cm,
+  y: 1.75cm,
+))
+```
+
+Typst will assume that documents written in Left-to-Right scripts are bound on
+the left while books written in Right-to-Left scripts are bound on the right.
+However, you will need to change this in some cases: If your first page is
+output by a different app, the binding is reversed from Typst's perspective.
+Also, some books, like English-language Mangas are customarily bound on the
+right, despite English using Left-to-Right script. To change the binding side
+and explicitly set where `inside` and `outside` are, set the `binding` argument
+in the `page` set rule.
+
+```example
+#set text(lang: "es")
+
+// Produce a book bound on the right,
+// even though it is set in Spanish.
+#set page(binding: right)
+```
+
+If `binding` is `left`, `inside` margins will be on the left on odd pages, and
+vice versa.
+
+## Add headers and footers
+Headers and footers are inserted in the top and bottom margin of every page. You
+can add custom headers and footers or just insert a page number.
+
+In case you need more than just a page number, the best way to insert a header
+and a footer are the `header` and `footer` arguments of the `page` set rule. You
+can pass any content as their values:
+
+```example
+#set page(header: [
+  _Lisa Strassner's Thesis_
+  #h(1fr)
+  National Academy of Sciences
+])
+```
+
+Headers are bottom-aligned by default, so that they do not collide with the top
+edge of the page. You can change this by wrapping your header in the `align`
+function.
+
+### Different header and footer on specific pages
+You'll need different headers and footers on some pages. For example, you may
+not want a header and footer on the title page. The example below shows how to
+conditionally remove the header on the first page:
+
+```example
+#set page(
+  header: locate(loc => {
+    if counter(page).at(loc).first() > 1 [
+      _Lisa Strassner's Thesis_
+      #h(1fr)
+      National Academy of Sciences
+    ]
+  }),
+)
+```
+
+This example may look intimidating, but let's break it down: We are telling
```

Comment:
This could also explain that you can have two different set rules for title page and secon page instead of the locate/if combo (at least if you have a separate title page without content flowing onto the next page).